dissect/target/helpers/certificate.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+import base64
+import binascii
+import hashlib
+from pathlib import Path
+
+from asn1crypto import pem, x509
```

Comment:
`asn1crypto` is not a base dependency (yet), so maybe put it in a guarded import while it's in a helper file.